src/components/entity_iid.rs

```diff
@@ -0,0 +1,57 @@
+use bevy::prelude::*;
+
+use std::borrow::Cow;
+
+/// [Component] added to all [LdtkEntity]s by default.
+///
+/// The `iid` stored in this component can be used to uniquely identify LDtk entities within an [LdtkAsset].
+#[derive(Clone, Debug, Default, Deref, DerefMut, Hash, Eq, PartialEq, Component, Reflect)]
```

Comment:
I prefer to not provide mutable access to the internal type here 
```suggestion
#[derive(Clone, Debug, Default, Deref, Hash, Eq, PartialEq, Component, Reflect)]
```